src/services/statistics.service.ts

```diff
@@ -77,7 +84,12 @@ export default class StatisticsService {
     try {
       const qb = this.appointmentRepository
         .createQueryBuilder('appointment')
-        .select('COALESCE(SUM(appointment.total_price), 0)', 'totalearnings');
+        .select(
+        `COALESCE(SUM(appointment.total_price + 
+                      COALESCE(appointment.tipAmount, 0) + 
```

Comment:
I see in a method below that to get the total tips you're joining through treatments, but here you're not using the query, is that right?, same to calculate the sales amount